plugins/rapids_plugin/notebooks/10_streamz.ipynb.ipynb

```diff
@@ -4293,7 +4293,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.7"
+   "version": "3.8.2"
```

Comment:
The streamz notebook mentions `self.required`. It should be removed. Remove this part:
https://github.com/rapidsai/gQuant/blob/ba348812d11f2c41671cc98e73a799d3d8444e1b/plugins/rapids_plugin/notebooks/10_streamz.ipynb.ipynb#L838-L843
```
        if nports > 2:
            self.required = {
                iport: {
                    'element': 'numbers'
                } for iport in iports_conn
            }
```

That required part of the code on additional ports is not used, which is good because it's not necessary.

Reply:
done